### PR TITLE
Update 2024-02-29-third-party-cookies.md

### PR DESCRIPTION
**Pull request by edunham**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [ ] The content has been run through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Tech review request (developer advocate or domain expert)
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

File: _source/_posts/2024-02-29-third-party-cookies.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ image: blog/3pc/social.jpg
 type: awareness
 ---
 
+**NOTE**: In July, Google has given an update on their [3rd party cookie deprecation effort](https://developers.google.com/privacy-sandbox/cookies).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below
```
Comment (alisaduncan) [inline _source/_posts/2024-02-29-third-party-cookies.md]: Please ensure "third-party cookie" is used consistently.

Comment (alisaduncan) [reply _source/_posts/2024-02-29-third-party-cookies.md]: Also ensure punctuation (missing period).

**Review by alisaduncan — CHANGES_REQUESTED** [review]

File: _source/_posts/2024-02-29-third-party-cookies.md (hunk 1 above)
Comment (edunham) [inline _source/_posts/2024-02-29-third-party-cookies.md]:
```suggestion
**NOTE**: In July, Google has given an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below.
```

File: _source/_posts/2024-02-29-third-party-cookies.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ image: blog/3pc/social.jpg
 type: awareness
 ---
 
+**NOTE**: In July, Google has given an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below.
```
Comment (alisaduncan) [inline _source/_posts/2024-02-29-third-party-cookies.md]:
```suggestion
**NOTE**: In July, Google provided an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/). Okta will work with Google to understand the specific impact on Okta customers. In the meantime, we continue to advise customers to migrate away from using 3rd party cookies, as outlined below.
```

**Review by alisaduncan — CHANGES_REQUESTED** [review]
Ran through Grammarly and noticed typos

File: _source/_posts/2024-02-29-third-party-cookies.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ image: blog/3pc/social.jpg
 type: awareness
 ---
 
+**NOTE**: In July, Google provided an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/). Okta will work with Google to understand the specific impact on Okta customers. In the meantime, we continue to advise customers to migrate away from using 3rd party cookies, as outlined below.
```
Comment (edunham) [inline _source/_posts/2024-02-29-third-party-cookies.md]:
```suggestion
**NOTE**: In July, Google provided an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/). Okta will work with Google to understand the specific impact on Okta customers. In the meantime, we continue to advise customers to migrate away from using third-party cookies, as outlined below.
```

**Review by alisaduncan — APPROVED** [review]

**Pull request merged**